### PR TITLE
#99 ScreenshotShortcut

### PR DESCRIPTION
**Pull request by HoloMoto**
## Overview

Allow screenshots to be taken using shortcuts based on #99.

　This change uses the ShortcutAttribute to register the CaptureScreenshotX() function from Editor/Utilities/ScreenshotUtilities.cs as a shortcut in Unity's ShortcutManager.

　This change allows developers to take a picture without opening the Unity menu.
　The purpose of this is to make it easier for xR app developers to take screensho

## Changes

Editor/Utilities/ScreenshotUtilities.cs

Added attribute [Shortcut("Graphics Tools/Take Screenshot X", (Key), (ShortcutModifiers))] to **CaptureScreenshotX()** function.

The shortcut assignment is Shift for alpha image and Alt key for normal screenshots.

　The shot is taken by simultaneously pressing the numeric key corresponding to the resolution at which the shot is to be taken.

　For example, to take a 4x shot that includes the alpha value, press Shift+4.

　The Shift key may interfere with the MRTK hand, which is debatable.
　I was initially going to assign the Ctrl key, but this conflicts with Unity's default operation of switching the display between the Game and Scene windows, so I chose the Shift key.

　This change automatically assigns the shortcut key to ShortcutManager.

 

## Additionally

　The separation of each function from MRTKv3 and the independence of MRGT as a graphics function has led me to use MRGT in my private as well as in my company's products by default.

　My colleagues have also given it high marks, saying that it is very easy to use.
　The function in the screenshot was also brought to my notice when I actually develop business applications, often while wearing an HMD.

　Making the screenshot function easier to use will be very helpful not only to us but also to many other developers.

　One of my colleagues said, "In a VR app, I'd like to have the ability to take 360-degree images! I think it would be very useful for many developers as well as us.
　I don't know if I can implement it with my current knowledge, but I would like to try it once this PR is done.

### CODE REVIEW

File: com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/ScreenshotUtilities.cs [hunk 1]
```diff
@@ -15,41 +16,45 @@ namespace Microsoft.MixedReality.GraphicsTools.Editor
...
 
+        [Shortcut("Graphics Tools/Take Screenshot 1x Alpha", KeyCode.Alpha1, ShortcutModifiers.Shift)]
         [MenuItem("Window/Graphics Tools/Take Screenshot/Native Resolution (Transparent Background)")]
         private static void CaptureScreenshot1xAlphaComposite()
         {
             CaptureScreenshot(GetScreenshotPath(), 1, true);
             EditorUtility.RevealInFinder(GetScreenshotDirectory());
         }
-
+        [Shortcut("Graphics Tools/Take Screenshot 2x", KeyCode.Alpha2, ShortcutModifiers.Alt)]
```
Comment (Cameron-Micka) [inline com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/ScreenshotUtilities.cs:34]: Nitpick: newline between the previous method and attribute:

```suggestion

        [Shortcut("Graphics Tools/Take Screenshot 2x", KeyCode.Alpha2, ShortcutModifiers.Alt)]
```

File: com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/ScreenshotUtilities.cs [hunk 2]
```diff
@@ -15,41 +16,45 @@ namespace Microsoft.MixedReality.GraphicsTools.Editor
...
 
+        [Shortcut("Graphics Tools/Take Screenshot 2x Alpha", KeyCode.Alpha2, ShortcutModifiers.Shift)]
         [MenuItem("Window/Graphics Tools/Take Screenshot/2x Resolution (Transparent Background)")]
         private static void CaptureScreenshot2xAlphaComposite()
         {
             CaptureScreenshot(GetScreenshotPath(), 2, true);
             EditorUtility.RevealInFinder(GetScreenshotDirectory());
         }
-
+        [Shortcut("Graphics Tools/Take Screenshot 4x", KeyCode.Alpha4, ShortcutModifiers.Alt)]
```
Comment (Cameron-Micka) [inline com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/ScreenshotUtilities.cs:49]: Nitpick: newline between the previous method and attribute:

```suggestion

        [Shortcut("Graphics Tools/Take Screenshot 4x", KeyCode.Alpha4, ShortcutModifiers.Alt)]
```

**Review by Cameron-Micka — APPROVED** [review]
Tried it out and worked wonderfully! Thank you. 🙏

**Comment by Cameron-Micka** [comment]
> One of my colleagues said, "In a VR app, I'd like to have the ability to take 360-degree images! I think it would be very useful for many developers as well as us.
　I don't know if I can implement it with my current knowledge, but I would like to try it once this PR is done.

Offhand I'm not sure how to do that either but sounds like a great feature to have. 😊

**Comment by HoloMoto** [comment]
Thank you! @Cameron-Micka 
I fixed the space location you pointed out!

**Pull request merged**